### DISCUSSION TITLE
Issue when setting profile to `max-perf`

### DISCUSSION THREAD

**Discussion by anotherfrontendguy** [ibhagwan/fzf-lua#1607] [Q&A] [answered]
Hey ✌️

thank you for your work on this plugin! I'm one of the "LazyVim Wave" adopters and loving it! (I hope the influx of questions isn't too crazy 😅)

I'm having a weird issue I cannot figure out - the best way to describe it can be found in the https://github.com/LazyVim/LazyVim/discussions/5200 discussion.

I don't even know if this is a `fzf-lua` issue in the end. If not sorry for the noice but maybe you can guide me in the right direction.

Thanks!

**Comment by ibhagwan** [comment, marked as answer]
https://github.com/LazyVim/LazyVim/discussions/5200#discussioncomment-11621729
